.github/workflows/build-chart.yaml

```
@@ -0,0 +1,29 @@
+name: Release Charts
+
+on:
+  push:
+
+permissions: write-all
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)